src/main/java/org/mule/extension/api/introspection/declaration/fluent/BaseDeclaration.java

```diff
@@ -36,52 +41,46 @@ public BaseDeclaration(String name)
     }
 
     /**
-     * Returns a map with the currently set model properties. Notice
-     * that this map is mutable and not thread-safe.
+     * Returns a {@link Set} with the currently added properties. Notice
+     * that this {@link Set} is mutable and not thread-safe.
```

Comment:
no. This is the fluent api used to describe the extension. This is not the same as the actual model which is then created after this